helm/examples/local/README.md

```diff
@@ -17,22 +17,22 @@ Examples of use:
 
   With a host directory of c:\hpccdata
 
-    helm install hpcc-localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
+    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
```

Comment:
this is correct. Just noting 'localfile' is the arbitrary instance name, it can be anything, but we want it to be "localfile" to match the examples values-localfile.yaml file. 
Whereas the chart name 'hpcc-localfile' cannot be changed (in this case it's pointed to be the path, but can be named/pointed in the repository).